scripts/patchseq_classic_metadata.qmd

```diff
@@ -0,0 +1,118 @@
+---
+title: "Generating pilot metadata"
+format: html
+---
+
+```{python}
+import numpy as np
+import pandas as pd
+import scanpy as sc
+```
+
+Read in metadata file from BIL dataset
+```{python}
+
+meta = pd.read_csv('/Users/jlchang/Documents/Broad-SCP/cb_work/2024/BRAIN_pilot_dataset/BIL/m1_patchseq_meta_data.csv', index_col=0, sep='\t')
```

Comment:
should this file be local to the PC or relative path to "./tests/data/annotation/metadata/convention/brain_rf1/patchseq_classic_metadata_test_10.tsv" file

Reply:
Good point. updated `912ffa6`